tests/integration/kubernetes/gha-run.sh

```diff
@@ -169,6 +169,11 @@ function run_tests() {
 	[ "$platform" = "kcli" ] && \
 		export KUBECONFIG="$HOME/.kcli/clusters/${CLUSTER_NAME:-kata-k8s}/auth/kubeconfig"
 
+	# Enable auto-generated policy for CI images that support policy.
+	#
+	# TODO: enable testing auto-generated policy for other types of hosts too.
+	[ "${KATA_HOST_OS}" = "cbl-mariner" ] && export AUTO_GENERATE_POLICY="yes"
```

Comment:
I don't have a firm opinion about that but I think I would export `AUTO_GENERATE_POLICY="yes"` at the Github's workflow level. Other job configuration rules will be passed to the scripts via environment variables from the workflows, for example, https://github.com/kata-containers/kata-containers/blob/main/.github/workflows/run-kata-coco-tests.yaml#L45

I understand that moving the rules to the workflows it will be harder to test on pull request time. So perhaps keep it as is, running some days then switching?

Reply:
sounds reasonable.